<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionWindow</class>
 <widget class="QMainWindow" name="ActionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>100</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>200</horstretch>
    <verstretch>80</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>100</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Actions</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>200</horstretch>
      <verstretch>80</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>USE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>200</horstretch>
      <verstretch>80</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>KICK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>200</horstretch>
      <verstretch>80</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>PULL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>200</horstretch>
      <verstretch>80</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>BUILD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_left">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>200</horstretch>
      <verstretch>80</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>LEFT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_right">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>500</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>200</horstretch>
      <verstretch>80</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>RIGHT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_up">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>200</horstretch>
      <verstretch>80</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>UP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_down">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>450</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>200</horstretch>
      <verstretch>80</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>DOWN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>200</horstretch>
      <verstretch>80</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>SAY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>200</horstretch>
      <verstretch>80</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>POINT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>200</horstretch>
      <verstretch>80</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>PUSH</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>200</horstretch>
      <verstretch>80</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>CURE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>200</horstretch>
      <verstretch>80</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>STEAL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_open">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>200</horstretch>
      <verstretch>80</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>OPEN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_close">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>200</horstretch>
      <verstretch>80</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>CLOSE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>200</horstretch>
      <verstretch>80</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>TAKE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_17">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>200</horstretch>
      <verstretch>80</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>PUT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_18">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>200</horstretch>
      <verstretch>80</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>THROW</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_findMe">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>200</horstretch>
      <verstretch>80</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>FIND ME</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>100</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
